draft-ietf-tls-esni.md

```diff
@@ -1265,7 +1270,7 @@ for ExtensionType (defined in {{!RFC8446}}):
 
 1. encrypted_client_hello(0xff08), with "TLS 1.3" column values being set to
    "CH, EE", and "Recommended" column being set to "Yes".
-3. outer_extension(0xff04), with the "TLS 1.3" column values being set to "CH",
+3. outer_extensions(0xff04), with the "TLS 1.3" column values being set to "CH",
```

Comment:
Should the TLS 1.3 column be "" instead of "CH"? It's not *really* a ClientHello extension.

Reply:
I think so.